assets/sass/modules/_forms.scss

```diff
@@ -189,6 +189,40 @@
       right: 10px;
       opacity: 0.5;
     }
+
+    .password-toggle {
+
+      .button-off {
+        display: none;
+      }
+
+      .button {
+        right: 5px;
```

Comment:
Just checking - have you verified this works as expected in all the browsers we support? I.e. manually verified across latest Chrome/Safari/Firefox, and IE8 - IE11 + Edge? Normally we can do IE9+, but there's currently a bug with embedded browsers that means the osw code needs to render in IE8.